docs/2.7.0/docs/daml-finance/tutorials/lifecycling/intro.rst

```diff
@@ -0,0 +1,46 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Lifecycling tutorials
+#####################
+
+This section explains how to lifecycle instruments in Daml Finance. Each tutorial combines a step by
+step description of different workflows with supporting code.
+
+Tutorial code
+*************
+
+To follow the tutorials, you can install the ``finance-lifecycling`` project via the Daml assistant
+by executing the following commands -
+
+On Unix-based systems execute:
+
+.. code-block:: shell
+
+   daml new finance-lifecycling --template=finance-lifecycling
+   cd finance-lifecycling
+   ./get-dependencies.sh
+   daml studio
+
+On Windows-based systems execute:
+
+.. code-block:: shell
+
+   daml new finance-lifecycling --template=finance-lifecycling
+   cd finance-lifecycling
+   get-dependencies.bat
+   daml studio
+
+Note: this will only work once Daml SDK v2.7 has been released.
```

Comment:
Do we need this note?

Reply:
I think it is useful, since some external users are looking at this latest version of the docs because of the latest features.